client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -261,7 +287,7 @@ class CheckoutThankYouHeader extends PureComponent {
 		return (
 			<div className={ classNames( 'checkout-thank-you__header', classes ) }>
 				<div className="checkout-thank-you__header-icon">
-					<img src={ `/calypso/images/upgrades/${ svg }` } />
+					<img src={ `/calypso/images/upgrades/${ svg }` } alt="" />
```

Comment:
That's not really addressing the issue :)

Reply:
If you check the lint message, it says `empty string for decorative images`. :) Or we can reuse the `getHeading()`?

Reply:
Actually, makes no sense, as that would have 2x the same info. empty `alt` is what we want here.

Reply:
🤷‍♂️ I guess it makes sense.